Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -543,7 +556,7 @@ namespace AtomSampleViewer
 
     void SampleComponentManager::OnTick(float deltaTime, [[maybe_unused]] AZ::ScriptTimePoint time)
     {
-        m_imGuiFrameTimer.PushValue(deltaTime * 1000.0f);
+        m_imGuiFrameTimer->PushValue(deltaTime * 1000.0f);
```

Comment:
Now that this is a pointer, it should either be checked before use

Reply:
done